### DISCUSSION TITLE
Minimum version required osx?

### DISCUSSION THREAD

**Discussion by tonicofonico** [streamlink/streamlink-twitch-gui#948] [Q&A] [answered]
What is the minimum version required for osx??
in plist is 10.13 but when i run:
````
$ ./nwjs 
dyld: lazy symbol binding failed: Symbol not found: ____chkstk_darwin
  Referenced from: /Applications/Streamlink Twitch GUI.app/Contents/Frameworks/nwjs Framework.framework/Versions/105.0.5195.102/libnode.dylib (which was built for Mac OS X 10.15)
  Expected in: /usr/lib/libSystem.B.dylib

dyld: Symbol not found: ____chkstk_darwin
  Referenced from: /Applications/Streamlink Twitch GUI.app/Contents/Frameworks/nwjs Framework.framework/Versions/105.0.5195.102/libnode.dylib (which was built for Mac OS X 10.15)
  Expected in: /usr/lib/libSystem.B.dylib
````

**Comment by bastimeyer** [comment, marked as answer]
See
https://github.com/nwjs/nw.js/issues/7946

This is apparently caused by the bump to NodeJS 18 in NW.js 0.64.1, and it requires at least macOS 10.15 Catalina.
https://nwjs.io/blog/

I can't help here, because this project is using the pre-built binaries provided by the NW.js maintainers, and the same NW.js version is used by all three supported platforms. The currently used version is 0.68.1.

The NW.js maintainers should've checked for any breaking changes in that release and made an according comment, but they did not, so I was unaware.

Either way, there is always a need for using more recent versions of NW.js and Chromium for security/bugfix reasons, so old platforms will become unsupported at some point eventually. Windows 7 and 8 will be the next one when the next Chromium version and thus NW.js gets released in February. Whether I will need to bump to this NW.js version in the next Twitch GUI release remains to be seen.
